Homework-6

Presnia opened the pull request.

Commit:
The class Firebase is created as well as the context directory with the testContext file

Commit:
The file firebaseContext is created 

The file firebaseContext is created and deleted all extra files used in test

Commit:
The bug with adding a new pokemon is solved

Commit:
Recreated the new structure, the styles is modified

ZarMarathon commented on src/routes/GamePage/routes/Start/index.js:
```diff
@@ -0,0 +1,84 @@
+                           values={values}
+                           active={active}
+                           minimize={minimize}
+                           clickOn={handleChangeActive}/>)
```
Номер раз!

ZarMarathon replied:
А в самом PokemonCard у тебя и в помине нет пропса clickOn

Commit:
Card styles have been adjusted, added the pokemonContext file

Created new routes

Commit:
The pokemon card styles are corrected, the handler is modified

Commit:
Added the method pokemonsContext.onSelectedPokemons to the handler

Handler for cards

Commit:
Changed behavior of the handleChangeSelected handler, added styles

Commit:
Rerouting have been done, added a handler to the button Start Game

Changed the hooks from useRouteMatch to useLocation. To the BoardPage are added pokemons from context

ZarMarathon reviewed (approved):
аппрув, жду 7е
